lara-typescript/src/page-item-authoring/common/components/interactive-authoring.tsx

```diff
@@ -48,6 +48,17 @@ export const InteractiveAuthoring: React.FC<Props> = (props) => {
     version: 1,
     error: null,
     mode: "authoring",
+    hostFeatures: {
+      modal: {
+        version: "1.0.0",
+        dialog: false,
+        lightbox: true,
+        alert: true
+      },
```

Comment:
I don't see new code supporting modals during authoring, was this supported before and just not added to the hostFeatures?

Reply:
I removed this, because I didn't see any modal support during authoring.